pkg/commands/plan/plan_run.go

```diff
@@ -285,49 +332,45 @@ func (c *PlanRunCommand) handleTerraformPlan(ctx context.Context) (*RunResult, e
 		lockfileMode = "readonly"
 	}
 
-	c.actions.Group("Initializing Terraform")
-
-	if _, err := c.terraformClient.Init(
-		ctx,
-		c.Stdout(),
-		multiStderr,
-		&terraform.InitOptions{
-			Input:       util.Ptr(false),
-			NoColor:     util.Ptr(true),
-			Lockfile:    util.Ptr(lockfileMode),
-			LockTimeout: util.Ptr(c.flagLockTimeout.String()),
-		}); err != nil {
+	_, err := c.withOutGroup("Initializing Terraform",
+		func() (int, error) {
+			return c.terraformClient.Init(ctx, c.Stdout(), multiStderr, &terraform.InitOptions{ //nolint:wrapcheck // Want passthrough
+				Input:       util.Ptr(false),
+				NoColor:     util.Ptr(true),
+				Lockfile:    util.Ptr(lockfileMode),
+				LockTimeout: util.Ptr(c.flagLockTimeout.String()),
+			})
+		},
+	)
+	if err != nil {
```

Comment:
Prefer cuddling:

```suggestion
	if _, err := c.withOutGroup("Initializing Terraform",
		func() (int, error) {
			return c.terraformClient.Init(ctx, c.Stdout(), multiStderr, &terraform.InitOptions{ //nolint:wrapcheck // Want passthrough
				Input:       util.Ptr(false),
				NoColor:     util.Ptr(true),
				Lockfile:    util.Ptr(lockfileMode),
				LockTimeout: util.Ptr(c.flagLockTimeout.String()),
			})
		},
	); err != nil {
```

Reply:
this seemed ugly to me, i refactored a bit to hopefully help